fossunited/chapters/doctype/foss_event_rsvp_submission/foss_event_rsvp_submission.py

```diff
@@ -58,6 +60,27 @@ def get_max_count(self):
         max_count = frappe.db.get_value(EVENT_RSVP, self.linked_rsvp, "max_rsvp_count")
         return max_count
 
+    def handle_submission_status(self):
+        # Check if the RSVP is accepting all incoming responses
+        is_accepting_all = bool(
+            frappe.db.get_value(EVENT_RSVP, self.linked_rsvp, "accept_all_incoming")
+        )
+
+        # If the RSVP is accepting all incoming responses, set the status to accepted
+        if is_accepting_all:
+            self.status = "Accepted"
+            return
```

Comment:
based on my earlier naming suggestion, this will read like

```suggestion
        submissions_need_approval = bool(
            frappe.db.get_value(EVENT_RSVP, self.linked_rsvp, "submissions_need_approval")
        )

        # If the RSVP is accepting all incoming responses, set the status to accepted
        if not submissions_need_approval:
            self.status = "Accepted"
            return
```

Reply:
Changing this to `Require Submission Approval`